README.md

```diff
@@ -31,4 +31,4 @@ There is documentation published at <https://quarkus.io> (docs' [sources are her
 
 * [Maven 3.5.3+](https://maven.apache.org/install.html)
 * [Java - OpenJDK 1.8+](https://adoptopenjdk.net/)
-* [GraalVM rc14+](https://www.graalvm.org/)
+* [GraalVM rc15+](https://www.graalvm.org/)
```

Comment:
It's going to be pretty hard to keep this value up to date.
The recommended version is documentation on the web site. Maybe adding a link to the web site would be ok. 
@gsmet WDYT?

Reply:
Why not have a general requirements page that goes over all of the dependency requirements including the os level ones as I have to track this down whenever I setup a new system. Having that as an initial link in every guide would ensure consistency.

Reply:
> It's going to be pretty hard to keep this value up to date.

Totally agree with you. 

> Why not have a general requirements page that goes over all of the dependency requirements including the os level ones as I have to track this down whenever I setup a new system. Having that as an initial link in every guide would ensure consistency.

+1

Reply:
Agreed.

Reply:
> The recommended version is documentation on the web site. Maybe adding a link to the web site would be ok.

Is this link that we are referring? https://quarkus.io/guides/building-native-image-guide